pallets/loans-ref/src/types.rs

```diff
@@ -700,7 +700,7 @@ impl<T: Config> ActiveLoan<T> {
 	}
 }
 
-#[cfg(any(test, feature = "runtime-benchmarks"))]
+#[cfg(any(feature = "std", feature = "runtime-benchmarks"))]
```

Comment:
Why do we need test utils under standard feature? This also reaches the node build then.

Reply:
For the pool tests? Then I would like to have a test feature flag

Reply:
The wasm is not built with `std`. It is used the `integration-test` to build `LoanInfo`. I've followed a similar schema as found in the system pallet for `std_block_number()` and other utilities there.

Reply:
In theory nodes can also use native runtimes. So it can hit your application. 

Just thinking clear features here do not hurt. But won't block.

Reply:
I was reading about it, and it seems like your node has a native binary and the wasm binary (that is what it will be on-chain). The native binary is part of your executable but the content is not stored on-chain, so these methods only bloat the local executable. But I think that does not add damage. And very probably, there are not even finally added because there are not used in the real node (only by the `integration-test` node), as 95% of the substrate code that is there but not used.

If I still do not convince you 😛, which feature could I add there? without creating a new one.